R/interval.R

```diff
@@ -0,0 +1,46 @@
+#' @title Interval
+#' @docType class
+#' @description  Simulation Interval (typically associated with an instace of \code{OutputSchema})
+#'
+#' @field startTime Start time of interval (instance of \code{Parameter})
+#' @field endTime End time of interval (instance of \code{Parameter})
+#' @field resolution Resolution of interval in pts/min (instance of \code{Parameter})
+#' @format NULL
+Interval <- R6::R6Class(
+  "Interval",
+  inherit = Container,
+  active = list(
+    startTime = function(value) {

```

Comment:
Start time, EndTime etc. .are actual instance of Parmaeter. One can do whatever we want we those parameters (update, value etc..)